app/refactor/viewmodels/p5/tad/Table4ViewModel.scala

```diff
@@ -35,8 +35,18 @@ object Table4ViewModel {
   def apply(ie029: CC029CType): Table4ViewModel =
     new Table4ViewModel(
       countryOfRoutingOfConsignment = ie029.Consignment.CountryOfRoutingOfConsignment.map(_.asString).semiColonSeparate.take30,
-      customsOfficeOfTransitDeclared = ie029.CustomsOfficeOfTransitDeclared.map(_.asString).semiColonSeparate.take60,
-      customsOfficeOfExitForTransitDeclared = ie029.CustomsOfficeOfExitForTransitDeclared.map(_.asString).semiColonSeparate.take60,
+      customsOfficeOfTransitDeclared = ie029.CustomsOfficeOfTransitDeclared
+        .map(
+          x => s"${x.sequenceNumber}/${x.referenceNumber}"
+        )
+        .semiColonSeparate
+        .take60,
+      customsOfficeOfExitForTransitDeclared = ie029.CustomsOfficeOfExitForTransitDeclared
+        .map(
+          x => s"${x.sequenceNumber}/${x.referenceNumber}"
+        )
+        .semiColonSeparate
+        .take60,
```

Comment:
should this not be a method which takes 9?

Reply:
@Reece-Carruthers I believe 9 is number of maximum occurrences which is already validated elsewhere eg:
cco15.xsd
    </xs:element>
      <xs:element name="CustomsOfficeOfTransitDeclared" minOccurs="0" maxOccurs="9" type="CustomsOfficeOfTransitDeclaredType04">
        <xs:annotation>
          <xs:documentation>
            <description value="CUSTOMS OFFICE OF TRANSIT (DECLARED)" />
            <optionality value="D" />
          </xs:documentation>
        </xs:annotation>
      </xs:element>
      <xs:element name="CustomsOfficeOfExitForTransitDeclared" minOccurs="0" maxOccurs="9" type="CustomsOfficeOfExitForTransitDeclaredType02">
        <xs:annotation>
          <xs:documentation>